### PR TITLE
implemented pagination of searched lectures

### PR DESCRIPTION
**Pull request by Tommybro**
one page's limit of the number of searched lectures = 6
image

### CODE REVIEW

**Comment by holenet** [comment]
Could upload a screenshot when a user hasn't searched yet?

File: src/store/ttrs/selectors.js [hunk 1]
```diff
@@ -51,6 +51,7 @@ export const initialTimeTable = {
 
 export const initialSearch = {
   lectures: [],
+  count: null,
```
Comment (holenet) [inline src/store/ttrs/selectors.js]: I think `0` is more appropriate for `count` than `null`.

**Comment by Tommybro** [comment]
image

**Comment by holenet** [comment]
And could you show a screenshot when the searched result has very many lectures?; ex) searching with empty parameter

File: src/components/molecules/SearchLecture/index.js [hunk 2]
```diff
@@ -63,6 +74,19 @@ class SearchLecture extends React.Component {
...
           <Modal.Actions>
+            {`${this.props.count}` &&
+            <div align="center">
+              <Pagination
+                ellipsisItem={{ content: <Icon name="ellipsis horizontal" />, icon: true }}
+                firstItem={{ content: <Icon name="angle double left" />, icon: true }}
+                lastItem={{ content: <Icon name="angle double right" />, icon: true }}
+                prevItem={{ content: <Icon name="angle left" />, icon: true }}
+                nextItem={{ content: <Icon name="angle right" />, icon: true }}
+                totalPages={parseInt((this.props.count - 1) / limit) + 1}
```
Comment (holenet) [inline src/components/molecules/SearchLecture/index.js]: Why did you use `parseInt`?

**Comment by holenet** [comment]
@Tommybro 
I think you've uploaded the wrong picture.

File: src/store/ttrs/selectors.js (hunk 1 above)
Comment (Tommybro) [reply src/store/ttrs/selectors.js]: I agree

**Comment by Tommybro** [comment]
image
image

File: src/components/molecules/SearchLecture/index.js [hunk 3]
```diff
@@ -33,6 +33,7 @@ class SearchLecture extends React.Component {
 
   handleClose = () => {
     this.setState({ 'course.name.abbrev': '' })
+    this.setState({ page: 1 })
```
Comment (holenet) [inline src/components/molecules/SearchLecture/index.js]: Please merge these two `this.setState` to just one statement.

**Pull request merged**